src/ui/header/AvHeader/AvHeader.vue

```diff
@@ -232,7 +232,7 @@ provide(registerNavigationLinkKey, () => hideModal)
     class="fr-header"
   >
     <div class="fr-header__body">
-      <div class="fr-container  width-inherit">
+      <div class="fr-container av-container  width-inherit">
```

Comment:
```suggestion
      <div class="fr-container  width-inherit">
```

J'ai finalement trouvé plus simple, ajoute ça en bas de `src/ui/styles/main.scss`

```vue
.fr-container {
  max-width: 90rem !important;
}
```